src/librustdoc/lib.rs

```diff
@@ -603,6 +603,7 @@ fn opts() -> Vec<RustcOptGroup> {
         unstable("show-type-layout", |o| {
             o.optflag("", "show-type-layout", "Include the memory layout of types in the docs")
         }),
+        stable("nocapture", |o| o.optflag("", "nocapture", "don't capture stdout/stderr of tests")),
```

Comment:
Can you make this unstable for now? I would rather avoid adding insta-stable options.

Reply:
Also, I think it would be a bit clearer to use "and" instead of "/":

```suggestion
        stable("nocapture", |o| o.optflag("", "nocapture", "Don't capture stdout and stderr of doctests")),
```

(I also capitalized for consistency with the other flags' descriptions.)

Reply:
Done!